docs/governance/templates/release_issue.md

```diff
@@ -73,6 +75,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
     (`helm install --namespace=agones-system agones agones/agones --version={version}-rc` if release candidate) and a smoke test to confirm everything is working.
 - [ ] Attach all assets found in the `release` folder to the draft Github Release.
 - [ ] If release candidate check the pre-release box on the draft Github Release
+- [ ] Copy any review changes to the release blog post are copied into the draft Github release.
```

Comment:
"Ensure that any review changes ..." or "Copy any review changes into the draft ..."

Reply:
Ermagerd that sentence makes no sense. Thanks.